simpledotorg/simple-server
# Localize regn, followup, and overdue child region names

**Story card:** TBD

I've absorbed #3745 into this PR for speeeeeeed.

## Because

We noticed that we were not localizing child region types in the tables on the "Details" page of reports. We notcied now that we're moving those tables to the Overview page.

## This addresses

Localizes the child region names (eg. "Division" in Bangladesh) on all reports tables.

## Test instructions

Set your country to Bangladesh, ensure that the child region names are localized.

<img width="1296" alt="Screen Shot 2022-05-20 at 3 25 43 PM" src="https://user-images.githubusercontent.com/4241399/169518225-f72e18c0-5209-4a27-afb6-c7e25a82334c.png">
